switchlink/sai/switchlink_handle_lag.c

```diff
@@ -172,6 +201,14 @@ static int create_lag_member(
   attr_list[ac].value.u32 = port_id;
   ac++;
 
+  // this SAI attribute is used to propagate oper state to switchapi module.
+  attr_list[ac].id = SAI_LAG_MEMBER_ATTR_EGRESS_DISABLE;
+  if (lag_member_intf->oper_state == IF_OPER_DOWN)
```

Comment:
if the link partner LACP member goes down, what event or change will we see on our DUT?
which parameter changes in the netlink notifications, so that we send pkts only via other member

Reply:
if the link partner LACP member goes down, we don't see any netlink events coming to the DUT and there is no change in the programming of tables.